computer-vision/README.md

```diff
@@ -0,0 +1,42 @@
+# Computer Vision
+
+The repository shall focus on the traditional aspects of computer vision, particularly multiview geometry. 
+
+## Pre - Requisites
```

Comment:
Image Processing is the main prerequisite. Move to CV after having an overview of IP.